### ISSUE TITLE
Shortcut to open "Doorhanger" can be seen as a written command

### ISSUE THREAD

**Issue by alexandra-martin** [mozilla/firefox-voice#238]
### Prerequisites:
Have mic permissions enabled.

### STR:
1. Press "Command-." on Mac or "Ctrl-." on Windows.
2. Observe the "Doorhanger".

### Expected result: 
The "Doorhanger" opens and a vocal or written command can be made.

### Actual result:
If the shortcut is pressed slower, the "Doorhanger" considers the pressed keys as a written command and displays the "Sorry, there was an issue" text like in [222](https://github.com/mozilla/firefox-voice/issues/222).

### Notes:
Reproduced on Mac 10.14.6 and Win10x64 with Firefox Nightly 71.0a1 (64-bit).
Shortcut keys need to be released pretty early, before the "Doorhanger" is even displayed.

![voice 29](https://user-images.githubusercontent.com/54401091/64777525-08586480-d563-11e9-8e5a-d6848cfb11f6.gif)

**Comment by ianb** [comment]
I *think* I figured this out, but I'm not sure. What I saw made the text input appear, but never actually inputed any text.

Also I guessed the same problem might occur for Windows, and tried to handle it (specifically the Ctrl key firing an event), but I couldn't verify.

**Comment by alexandra-martin** [comment]
The bug is still reproduced on Mac 10.14.6 and Win10x64 with Firefox Nightly 71.0a1 (64-bit). Indeed the "Cmd"/"Ctrl" key may play a part in this bug, as it is the first key pressed and the last key released for the shortcut. In order for the text input to not be displayed the "Cmd"/"Ctrl" key needs to be release at the same time with the "." key.

**Comment by ianb** [comment]
I believe #278 fixes this (better than the last fix)

**Comment by alexandra-martin** [comment]
Verified on Mac 10.14.6 and Win10x64 with Firefox Nightly 71.0a1 (64-bit). 
The only issue I found is that if "." key is pressed longer, the text input will appear with several dots, like in the second gif. But that seems too extreme to me, that a user will keep the keys pressed for so long, after the chime and it seems normal that the "Doorhanger" decides to display the text input. Maybe I am wrong, but wanted to let you know.

![voice 43](https://user-images.githubusercontent.com/54401091/65310720-9fdb3a00-db97-11e9-86f9-2d4325d605c5.gif)

![voice 44](https://user-images.githubusercontent.com/54401091/65311388-e715fa80-db98-11e9-9036-0573b4d36aae.gif)
